gcloud_bigtable/google/bigtable/admin/cluster/v1/bigtable_cluster_service_api.py

```diff
@@ -0,0 +1,262 @@
+# This file was generated from google/bigtable/admin/cluster/v1/bigtable_cluster_service.proto,
+# and updates to that file get reflected here through a regular refresh
+# process. However, manual and updates to that file get reflected here
+# through a regular refresh additions are allowed because the refresh
+# process performs a 3-way merge in order to preserve those manual additions.
+# In order not to break the refresh process, only certain types of
+# modifications are allowed.
+#
+# Allowed modifications:
+# 1. New methods (these should be added to the end of the class)
+# 2. "Notes specific to this wrapper method" sections in the method
+#    documentation
+#
+# Happy editing!
+
+from google.bigtable.admin.cluster.v1 import bigtable_cluster_data_pb2
+from google.bigtable.admin.cluster.v1 import bigtable_cluster_service_messages_pb2
+from google.bigtable.admin.cluster.v1 import bigtable_cluster_service_pb2
+from google.gax import api_callable
+from google.gax import api_utils
+from google.gax import page_descriptor
+from google.longrunning import operations_pb2
+from google.protobuf import timestamp_pb2
+
+class BigtableClusterServiceApi(object):
+  """Service for managing zonal Cloud Bigtable resources."""
+
+  # The default address of the logging service.
+  _SERVICE_ADDRESS = "bigtableclusteradmin.googleapis.com"
+
+  # The default port of the logging service.
+  _DEFAULT_SERVICE_PORT = 443
+
+  # The scopes needed to make gRPC calls to all of the methods defined in
+  # this service
+  _ALL_SCOPES = [
+      'https://www.googleapis.com/auth/bigtable.admin',
+      'https://www.googleapis.com/auth/bigtable.admin.cluster',
+      'https://www.googleapis.com/auth/cloud-bigtable.admin',
+      'https://www.googleapis.com/auth/cloud-bigtable.admin.cluster',
+      'https://www.googleapis.com/auth/cloud-platform'
+  ]
+
+  def __init__(
+      self, service_path=_SERVICE_ADDRESS, port=_DEFAULT_SERVICE_PORT,
+      channel=None, ssl_creds=None, scopes=_ALL_SCOPES,
+      is_idempotent_retrying=True, max_attempts=3, timeout=30):
+
+    self.defaults = api_callable.ApiCallableDefaults(
+      timeout=timeout, max_attempts=max_attempts,
+      is_idempotent_retrying=is_idempotent_retrying)
+
+    self.stub = api_utils.create_stub(
+      bigtable_cluster_service_pb2.beta_create_BigtableClusterService_stub, service_path, port,
+      ssl_creds=ssl_creds, channel=channel, scopes=scopes)
+
+  def __enter__(self):
+    return self
+
+  def __exit__(self, type, value, traceback):
+    self.close()
+
+  def close(self):
+    del self.stub
+
+  # Properties
+  @property
+  def channel(self):
+    return self.channel
+
+  # Page descriptors
+
+  # Service calls
+  def list_zones(self, name="", **kwargs):
+    """Lists the supported zones for the given project."""
+    list_zones_request = bigtable_cluster_service_messages_pb2.ListZonesRequest(
+        name=name, **kwargs)
+    return self.list_zones_callable()(list_zones_request)
```

Comment:
As mentioned below, this could just be

```python
    return api_callable.idempotent_callable(
        self.stub.ListZones,
        is_retrying=None,
        max_attempts=None,
        defaults=self.defaults)
```

and the generator would have no problem doing the mapping

```python
list_zones --> self.stub.ListZones
```